cdi/3.0/_index.md

```diff
@@ -1,26 +1,30 @@
 ---
-title: "Jakarta Context Dependency Injection 3.0 (milestone)"
-date: 2020-05-28
+title: "Jakarta Context Dependency Injection 3.0"
+date: 2020-08-26
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
 * [Jakarta Contexts Dependency Injection 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cdi/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Contexts Dependency Injection 3.0 Specification Document](./cdi-spec-3.0.0-M4.pdf) (PDF)
-* [Jakarta Contexts Dependency Injection 3.0 Specification Document](./cdi-spec-3.0.0-M4.html) (HTML)
+* [Jakarta Contexts Dependency Injection 3.0 Specification Document](jakarta-cdi-spec-3.0.pdf) (PDF)
+* [Jakarta Contexts Dependency Injection 3.0 Specification Document](jakarta-cdi-spec-3.0.html) (HTML)
 * [Jakarta Contexts Dependency Injection 3.0 Javadoc](./apidocs)
-* [Jakarta Contexts Dependency Injection 3.0 TCK](http://download.eclipse.org/ee4j/cdi/cdi-tck-3.0.0-M3-dist.zip)([sig](),[sha](),[pub]())
-  * TCK Summary Results [Weld 4.0.0.Alpha2](./TCK-summary.html)
+* [Jakarta Contexts Dependency Injection 3.0 TCK](http://download.eclipse.org/jakartaee/cdi/3.0/cdi-tck-3.0.0-dist.zip)
+([sig](http://download.eclipse.org/jakartaee/cdi/cdi-tck-3.0.0-dist.zip.sig),
+[sha](http://download.eclipse.org/jakartaee/cdi/cdi-tck-3.0.0-dist.zip.sha256),
+[pub](https://github.com/jakartaee/specification-committee/blob/master/jakartaee-spec-committee.pub))
 
 * Maven coordinates
-  * [jakarta.cdi:jakarta.cdi-api:jar:3.0.0-M4](https://repo1.maven.org/maven2/jakarta/enterprise/jakarta.enterprise.cdi-api/3.0.0-M4/)
+  * [jakarta.enterprise:jakarta.cdi-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.enterprise/jakarta.enterprise.cdi-api/3.0.0/jar)
 
+* Schemas
+  * [XML Schema for the CDI 3.0 beans.xml deployment](https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd)
```

Comment:
This schema does not seem to exist, either in Draft mode or Final...  I thought it was there on the previous review, but now I can't seem to find it.

Reply:
I can't find the 3.0 version of this XSD anywhere on our Jakarta EE Schema pages...
http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/beans_2_0.xsd